server/app/views/applicant/NorthStarAddressCorrectionBlockView.java

```diff
@@ -62,7 +62,7 @@ public String render(
     context.setVariable("pageTitle", pageTitle);
 
     context.setVariable("programTitle", params.programTitle());
-    context.setVariable("programDescription", params.programDescription());
+    context.setVariable("programDescription", params.programShortDescription());
```

Comment:
```suggestion
    context.setVariable("programShortDescription", params.programShortDescription());
```
(blocking) Please update the variable here and in the html.

Reply:
Done, thanks for the suggestion!

Reply:
Thank you!